### PR TITLE
Bug 1539566 - Request server version on about page load

### PR DESCRIPTION
**Pull request by spadgett**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1539566

/assign @jwforres

### CODE REVIEW

**Comment by spadgett** [comment]
/hold

Requires common PR: https://github.com/openshift/origin-web-common/pull/293

**Comment by spadgett** [comment]
@damemi FYI, I have a PR that changes the about page again (sorry)

@jwforres Let me know what you think of the build date. We could leave the template as-is and change the controller if we don't care about the build date.

File: app/views/about.html [hunk 1]
```diff
@@ -16,11 +16,17 @@ <h2>About</h2>
                 <h2 id="version">Version</h2>
                 <dl class="dl-horizontal left">
                   <dt>OpenShift Master:</dt>
-                  <dd>{{version.master.openshift || 'unknown'}}</dd>
+                  <dd>
+                    {{version.master.openshift.gitVersion}}
```
Comment (jwforres) [inline app/views/about.html]: we officially documented `version.master.openshift` and `version.master.kubernetes` in the about page customization.  I'd rather not change those scope variables unnecessarily.

**Comment by spadgett** [comment]
@jwforres updated, PTAL. I removed build date for now.

**Comment by spadgett** [comment]
cluster up failed

/retest

**Comment by spadgett** [comment]
@jwforres We might have a flake.

```
1) User adds a template to a project after creating a new project using the Import YAML tab should save the template in the project catalog
  - Error: Wait timed out after 5016ms
      at /data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:1425:29
    at /data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/goog/base.js:1582:15
    at webdriver.promise.ControlFlow.runInNewFrame_ (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:1654:20)
    at notify (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:465:12)
    at then (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:522:7)
    at Object.webdriver.promise.asap (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:767:11)
    at newFrame.onComplete (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:1670:25)
    at webdriver.promise.Frame_.notify_ (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:1914:5)
    at Timeout._onTimeout (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:1901:11)
    at ontimeout (timers.js:386:11)
==== async task ====
    at webdriver.promise.ControlFlow.wait (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:1410:15)
    at webdriver.WebDriver.wait (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/webdriver.js:646:21)
    at Object.exports.login (/data/src/github.com/openshift/origin-web-console/test/integration/helpers.js:35:10)
    at beforeEach (/data/src/github.com/openshift/origin-web-console/test/integration/features/user_adds_template_to_project.spec.js:16:7)
    at /data/src/github.com/openshift/origin-web-console/node_modules/jasminewd/index.js:94:14
    at webdriver.promise.ControlFlow.runInNewFrame_ (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:1654:20)
    at webdriver.promise.ControlFlow.runEventLoop_ (/data/src/github.com/openshift/origin-web-console/node_modules/selenium-webdriver/lib/webdriver/promise.js:1518:8)
    at ontimeout (timers.js:386:11)
    at tryOnTimeout (timers.js:250:5)
==== async task ====
Asynchronous test function: beforeEach()
    at .<anonymous> (/data/src/github.com/openshift/origin-web-console/node_modules/jasminewd/index.js:93:33)
    at .<anonymous> (/data/src/github.com/openshift/origin-web-console/node_modules/minijasminenode/lib/async-callback.js:45:37)
Error
    at describe (/data/src/github.com/openshift/origin-web-console/test/integration/features/user_adds_template_to_project.spec.js:14:3)
    at Object.<anonymous> (/data/src/github.com/openshift/origin-web-console/test/integration/features/user_adds_template_to_project.spec.js:12:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
  - timeout: timed out after 60000 msec waiting for spec to complete
```

/retest

**Comment by spadgett** [comment]
Rebased

**Comment by spadgett** [comment]
These test failures all look like flake #2105 (except cluster up failed one time)

**Pull request merged**